bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -132,8 +132,8 @@ spec:
                 server: <vCenter Server FQDN/IP>
     ```
     Example MachineSet for other cloud providers:
-    - [AWS](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md)
-    - [Azure](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-azure.md)
+    - [AWS](https://docs.openshift.com/container-platform/4.11/windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.html)
```

Comment:
Do we have a stance on what these links should be for the community vs Red Hat operator?

Reply:
I think we said community docs should link to our github repo, but @openshift/openshift-team-windows-containers please confirm

Reply:
> Do we have a stance on what these links should be for the community vs Red Hat operator?

Not sure if we discussed community doc links in the past. My two cents: 
- Links for  Red Hat operator should point to official [docs](https://docs.openshift.com/container-platform/latest/windows_containers/)
- Links for the community may also point to the `latest` official docs (less maintenance). GitHub may be fine too.

Reply:
We have been pointing to github docs for community operator and official docs for RH operator. 
Is there a reason for updating these links? trying to get a context.

Reply:
>We have been pointing to github docs for community operator and official docs for RH opera

This is the correct approach. We should not link the community CSV to the official docs. It is not about maintenance but about being able to release features quicker that won't have official docs.

Reply:
Agree

Reply:
Why are we updating this? Is this so that OpenShift docs are reflected in the Red Hat operator?

Reply:
To use OpenShift docs in the Red Hat operator CSV

Reply:
Then please call that out. Our PRs and commits should say WHAT is being done and WHY. I am noticing quite a few commits where the WHY is missing.

Reply:
Thanks for highlighting this.  Commit message updated.